docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -125,7 +125,7 @@ For the first two files in the list, the default mode will be determined by the
 parameter will not be sent to the file module while the final file will receive the `mode=0444` option.
 
 .. note:: If you are "chaining" additional filters after the `default(omit)` filter, you should instead do something like this:
-      `"{{ foo | default(None) | some_filter or omit }}"`. In this example, the default `None` (python null) value will cause the
+      `"{{ foo | default(None) | some_filter or omit }}"`. In this example, the default `None` (Python null) value will cause the
```

Comment:
```suggestion
      ``"{{ foo | default(None) | some_filter or omit }}"``. In this example, the default ``None`` (Python null) value will cause the
```
This is done wrong a lot in existing documentation, but since you touch this, please change it (also in the line above) :)